lib/datasource/repology/index.spec.ts

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs';
+import { DATASOURCE_FAILURE } from '../../constants/error-messages';
+import _got from '../../util/got';
+import * as ds from '.';
+
+jest.mock('../../util/got');
```

Comment:
Please don't mock `got`. Use `nock` tomock web requests.

Sample:

https://github.com/renovatebot/renovate/blob/f652a8c7d6cd3f7316d91f8f0f7f0246b7bb0a8e/lib/config/presets/github/index.spec.ts#L14-L19

https://github.com/renovatebot/renovate/blob/f652a8c7d6cd3f7316d91f8f0f7f0246b7bb0a8e/lib/config/presets/github/index.spec.ts#L171-L182

Reply:
Thanks for the hint! I wasn't aware of this possibility, however using `httpMock` is certainly much nicer than directly messing around with `got`. I've adjusted the test cases accordingly and also added the traces as test snapshots.